src/idvc/dvc_interface.py

```diff
@@ -1914,7 +1942,13 @@ def MaskWorker(self, type):
             
         self.create_progress_window("Loading", "Loading Mask")
         self.mask_worker.signals.progress.connect(self.progress)
-       
+
+        # disable tracing on the viewer
+        v.imageTracer.Off()
+        # set the button to unchecked and text to start tracing
+        self.mask_parameters['start_tracing'].setChecked(False)
+        self.mask_parameters['start_tracing'].setText("Start Tracing")    
+
```

Comment:
This code seems to try to turn tracing off after a mask has been made? But I find tracing is still on once my mask has been made 😞

Reply:
Now that I have tested more thoroughly I see that the only case where it doesn't turn off tracing and reset the button is the case where I already have a mask, then I trace and create a new mask.

If I have first cleared the mask, or it is a fresh run of the app and there is no previous mask displayed, then it resets the button and turns off the tracing just fine.

Reply:
It isn't a big issue, just something I noticed when testing...

Reply:
I'll check it!

Reply:
I tried as you say
> I already have a mask, then I trace and create a new mask.

My app first let me do it, but no result from my tracing. At a second attempt it crashed.

Reply:
Weird, I had no problems with crashing! Just that the tracing was/wasn't enabled at the right time

Reply:
I implemented a workaround with _the wise owl advise not to be naughty_